src/ci/docker/cross/install-mips-musl.sh

```diff
@@ -10,15 +10,10 @@
 
 set -ex
 
-mkdir /usr/local/mips-linux-musl
-
-# originally from
-# https://downloads.openwrt.org/snapshots/trunk/ar71xx/generic/
-# OpenWrt-Toolchain-ar71xx-generic_gcc-5.3.0_musl-1.1.16.Linux-x86_64.tar.bz2
+# This file was generated with https://github.com/richfelker/musl-cross-make and
+# http://www.nongnu.org/libunwind/. The cross compiler in the sysroot was
```

Comment:
Pretty sure you need the llvm libunwind, when initially adding musl support the nongnu one did not cope well - https://github.com/rust-lang/rust/pull/24777#issuecomment-100634662 (though that was over two years ago, it may be better now? I still think it'd be better to stick with the tried and tested LLVM one).

See https://github.com/rust-lang/rust/blob/631f761/src/ci/docker/dist-x86_64-musl/build-musl.sh#L28-L38